CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # qbic-pipelines/rnadeseq: Changelog
 
+## 1.4.1 - dev
+
+### Added
+- Optional parameter "--nsubgenes" to change the number of subset genes for vst
+- Added a line mentioning the --nsubgenes value in the report output. 
+
```

Comment:
I propose adding this rather under the point 1.4.0 - dev below.

Reply:
Thanks Daniel, I just made the changes.